buddyboss/buddyboss-platform
# When send only emoticons in messaging system, does not send

### All Submissions:






Fixes #1394 

https://github.com/buddyboss/buddyboss-theme/pull/963

### How to test the changes in this Pull Request:

Test this changes in:
Messages
Group Messages
Activity Post Update Editor
Forums Topic and Reply Editor

1. Go to Messages -> Try to send empty message and try to send only emoji.
2. It should display validation error message when sent empty message
3. It shouldn't display validation error message when send only emoji

### Proof Screenshots or Video



### Other information:
